project/plugins.sbt

```diff
@@ -0,0 +1,5 @@
+logLevel := Level.Warn
+
+addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0")
+
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.4")
```

Comment:
I quite liked using this release plugin at The Guardian so I thought I'd add it here too